.travis.yml

```diff
@@ -9,107 +10,59 @@ install:
   - pip install -r requirements.txt -r test-requirements.txt
   - pip install -U .
   - pip freeze
+before_script:
+  - "[ $NUMPY = latest ] && pip install --upgrade numpy || pip install numpy==$NUMPY"
+  - "[ $PANDAS = latest ] && pip install --upgrade pandas || pip install pandas==$PANDAS"
+  - "[ $SCIKITLEARN = latest ] && pip install --upgrade scikit-learn || pip install scikit-learn==$SCIKITLEARN"
+  - "[ $DASK = latest ] && pip install --upgrade dask || pip install dask==$DASK"
+  - "[ $DISTRIBUTED = latest ] && pip install --upgrade distributed || pip install distributed==$DISTRIBUTED"
+  # need to downgrade tornado manually
+  - "[ $SCIPY = latest ] || pip install tornado==4.5.3"
+  - "[ $SCIPY = latest ] && pip install --upgrade scipy || pip install scipy==$SCIPY"
+  - pip list
+  - sed -i -e 's/-n auto/-n 2/g' setup.cfg
+  - sed -e '/^\s*--cov tsfresh/d' setup.cfg > setup-nocov.cfg
+env:
+  global:
+    - PYTEST_ADDOPTS="-c setup-nocov.cfg"
```

Comment:
Maybe you want to add a comment why you do this here (and to the line above)

Reply:
Ok

Reply:
Maybe a simpler solution would be to run all tests without coverage during the build phase, and to run a single job with coverage afterwards.